packages/mobile/src/screens/edit-playlist-screen/EditPlaylistNavigator.tsx

```diff
@@ -0,0 +1,27 @@
+import type { EditPlaylistValues } from '@audius/common'
+import { createNativeStackNavigator } from '@react-navigation/native-stack'
+import type { FormikProps } from 'formik'
+
+import { useAppScreenOptions } from 'app/screens/app-screen/useAppScreenOptions'
+
+import { EditPlaylistForm } from './EditPlaylistForm'
+
+const Stack = createNativeStackNavigator()
+
+const screenOptionOverrides = { headerRight: () => null }
+
+export const EditPlaylistNavigator = (
+  props: FormikProps<EditPlaylistValues> & { playlistId: number }
+) => {
+  const screenOptions = useAppScreenOptions(screenOptionOverrides)
+
+  return (
+    <>
```

Comment:
I think in the case of this screen, we dont need a navigator since it is all one screen, so maybe we can remove this part of things? lmk if you want to pair on the component hierarchy here!

Reply:
this was the only way that the header and top nav things were showing up. I might be mistaken, but down to chat about this